assets/js/views/ChargingSessions.vue

```diff
@@ -228,6 +226,16 @@ export default {
 			const modal = Modal.getOrCreateInstance(document.getElementById("sessionDetailsModal"));
 			modal.show();
 		},
+		csvHrefLink(locale, month) {
```

Comment:
Wenn du hier statt `month` `group` oder `groupKey` als Namen verwendest ists nicht missverständlich in der Methode.

Generell bin ich rückwirkend etwas unzufrieden damit, dass ich die Monat/Jahr Info nur so indirekt über den Key in die Datenstruktur gepackt habe. Das Splitten ist nicht so elegant. Ich find das nicht schlimm, können wir mMn erstmal so lassen. Aber falls du nen Verbesserungsvorschlag hast der einfacher zu lesen ist gerne her damit :D

Reply:
Ja darüber bin ich auch gestolpert und habe ebenfalls eine Trennung nach Jahren gesucht. Ich wollte noch schauen, ob es schon fertige Vue Table UIs gibt die man als dependency Integrieren und anpassen kann. Was meinst du?

Reply:
Mit fertigen Libraries in der Größenordnung hab ich bislang keine guten Erfahrungen gemacht. Das ist am Anfang oft super weil man viel geschenkt bekommt, auf Dauer aber meist anstrengend weil wenig flexibel. Ich würd hier erstmal bei ner einfachen Lösung bleiben.

Generell hat die Seite aber noch viel Potential für Optimierung. Die ist damals als schneller Quickwin gebaut worden :)
Könnte mir auch gut vorstellen, dass wir hier grafischer werden und bspw. mit nem Balkendiagram oben arbeiten über das man auch selektieren kann.

Reply:
Sonst können wir dieses Feature wie geplant klein halten und dann würde ich mich an einen größeren Umbau (Filtern, Sortieren, Grafiken) im Folgenden kümmern. Dann könnten wir @naltatis die UI planen und ich setze das dann mit Rücksprache um. Was sagt ihr @andig & @naltatis ?

Reply:
Klar, gerne schrittweise- das hatte ich nicht mitbekommen.

Reply:
Genau richtig. Lass uns das Schritt für Schritt machen.